### PR TITLE
feat: Web 640 eng transaction states redesign

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-640?atlOrigin=eyJpIjoiYjNmMWY0MGI5MDdiNDVjYWE3YzhmOWY0YzI4NWNhNmUiLCJwIjoiaiJ9

figma: https://www.figma.com/file/kNSDBMpOzxSTOP6MerohLm/Web-App-Spec-(Phase-0-%26-Phase-1)-%5BWIP%5D?node-id=2225%3A50867

Screen Shot 2022-08-18 at 5 57 11 PM
Screen Shot 2022-08-18 at 5 54 38 PM
Screen Shot 2022-08-18 at 5 48 33 PM

### CODE REVIEW

File: src/components/WalletModal/index.tsx [hunk 1]
```diff
@@ -83,7 +83,11 @@ const HeaderRow = styled.div<{ redesignFlag?: boolean }>`
 
 const ContentWrapper = styled.div<{ redesignFlag?: boolean }>`
   background-color: ${({ theme, redesignFlag }) => (redesignFlag ? theme.backgroundSurface : theme.deprecated_bg0)};
+<<<<<<< HEAD
```
Comment (kayleegeorge) [inline src/components/WalletModal/index.tsx:86]: fix merge <<<< thingy

**Review by kayleegeorge — APPROVED** [review]
lgtm after comment :-)

oh also there's a period in 'transaction rejected' but in Figma it doesn't have one

File: src/components/WalletModal/index.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/components/WalletModal/index.tsx]: oof thanks for catching

**Comment by lynnshaoyu** [comment]
> lgtm after comment :-)
> 
> oh also there's a period in 'transaction rejected' but in Figma it doesn't have one

fixed ty!

**Comment by infredible** [comment]
lgtm

**Pull request merged**